data/dbus/meson.build

```diff
@@ -0,0 +1,15 @@
+configure_file(
+  input: 'org.gnome.GPaste.Ui.dbus.in',
+  configuration: conf,
+  output: 'org.gnome.GPaste.Ui.service',
+  install: true,
+  install_dir: dbus_service_dir,
+)
+
+configure_file(
+  input: 'org.gnome.GPaste.dbus.in',
+  configuration: conf,
```

Comment:
While at it, not sure if there's a mechanism in meson for this, but it would be great to make the systemd line conditional on the systemd option

Reply:
Meson only supports that via configure_file, I'll see how to do this elegantly tomorrow

Reply:
FWIW, I don't think this is required. Void Linux has lots of services which have SystemdService=* in their dbus file and it doesn't break anything as such as dbus isn't started in systemd mode.